### ISSUE TITLE
Decks marked as valid for casual despite new errata

### ISSUE THREAD

**Issue by Preachieved** [Null-Signal-Games/netrunnerdb#29]
Decks that have been made invalid by errata (ASPP one per deck) are marked as legal and valid, despite having three ASPP in them.
ex) https://netrunnerdb.com/en/decklist/32166/ctz-undefeated

**Comment by x3N1GM4x** [comment]
While it's probably not super relevant now that Astro has rotated, the errata is missing from the card, which I will log in netrunner-cards-json. 

I built a deck from a Collection of: Core Set, Genesis, C&C, Spin, H&P and Lunar, with 3 Astros and MWL 1.1 specified and was not prevented from adding 3x Astro or warned about the illegality until I had published the deck. Conversely, if I did the same with Philotic Entanglement, I was only allowed to add a single copy to my deck, despite there being 3 physical copies in H&P.
